packages/babel-cli/package.json

```diff
@@ -19,7 +19,7 @@
     "compiler"
   ],
   "dependencies": {
-    "commander": "^2.8.1",
+    "commander": "^4.0.1",
```

Comment:
@nicolo-ribaudo I don't think a major bump of dependencies is a breaking change as long as we have handled necessary breaking changes on our side, isn't it?